### DISCUSSION TITLE
Calibrating pressure setting for each actuator

### DISCUSSION THREAD

**Discussion by flexmatics** [nliaudat/esp32_8ch_motor_shield#25] [Q&A] [answered]
Hi,

I setup the shield with HmIP-VDMOT actuators and it works fine (incl. proper BEMF readings after replacing resistors to 10 ohm). To fine tune, I'd like to define exact pressure for each actuator so that it would be clear when actuator is engaged for example 25%, 50%, 75%. This way I could more or less understand water flow through each valve. I attach photos of system I have - these one are quite sensitive (i.e. slight pressure by actuator can fully close or open the valve).

Any idea / guidance how this could be achieved? Or only solution is to play around with bemf_trigger_x variable?

![image](https://user-images.githubusercontent.com/79056776/196519377-c41381c6-866e-4c84-9976-f405faa60280.png)
![image](https://user-images.githubusercontent.com/79056776/196519859-9bb4b2a2-333b-41b3-a77d-65a5f8160573.png)

**Comment by nliaudat** [comment, marked as answer]
Hi, 
I have no experience with pressure settings for floor heating and their real correlation with temperature rise. Sorry.

Bemf_trigger_x are valve endstops only and the valve position is calculated between the two endstops (timed based).

In fact, the exact valve position is not important at all cause it's recalculated at predifined interval (see time.yaml) to match the target temperature. 
I advise to not care the valve position, but if you want fine tune, you can play with "min movement change %" and set it to lower than 5% (default) and perhaps lower the check interval. 

That kind of floor heating have a lot of inertia and it could take 2 days to fully heat a room passing from closed to opened valve. That's why the only real value you must care is the target temperature (low+max)/2.

You also can setup the target temperature with [weather forecast in hassio](https://github.com/nliaudat/floor-heating-controller/wiki/Hassio-custom-climate-components)
